libs/ledger-live-common/src/__tests__/api/test-helpers/scenarios.ts

```diff
@@ -0,0 +1,219 @@
+type Event = [string] | [string, string] | [string, string, boolean]; // callback, payload, eager
```

Comment:
FYI you can create "named tuples":

```TS
type Event =
  | [callback: string]
  | [callback: string, payload: string]
  | [callback: string, payload: string, eager: boolean];
```

It should have the same behavior than normal tuples

Reply:
Oh nice!